indico/web/rh.py

```diff
@@ -233,8 +233,10 @@ def _do_process(self):
         if rv is not None:
             return rv
 
-        self._check_access()
-        signals.rh.check_access.send(type(self), rh=self)
+        signal_rv = values_from_signal(signals.rh.before_check_access.send(type(self), rh=self))
+        if len(signal_rv) == 0:
```

Comment:
IIRC we discussed the return value form the signal before and came up with something different than what's here? i don't really like that *any* return value will skip the original access check...

Reply:
I missed the message where you proposed something different:

> True -> skip access check; False -> raise Forbidden with a generic message (ice to have for the sake of completeness, most code should probably raise an error itself though); None -> do the usual access checks. if a mix of True and False is returned, consider it False. IIRC we do the same in the can_access override signal